src/index.js

```diff
@@ -1,39 +1,108 @@
+// Force sentry DSN into environment variables
+// In the future, will be set by the stack
+process.env.SENTRY_DSN =
+  process.env.SENTRY_DSN ||
+'https://0a734fc9bea84117bd562b823e8819e8:601dc1f6690449eca232c747daff7fac@sentry.cozycloud.cc/34'
+
 const {
   BaseKonnector,
   requestFactory,
   log,
-  saveFiles,
-  addData
+  errors,
+  saveBills
 } = require('cozy-konnector-libs')
-const request = requestFactory({ cheerio: true })
-
-const baseUrl = 'http://books.toscrape.com'
+let request = requestFactory()
+const jar = request.jar()
+request = requestFactory({
+  //debug: true,
+  cheerio: true,
+  jar: jar
+})
+const moment = require('moment')
+moment.locale('fr')
 
+const baseUrl = 'https://www.boulanger.com/webapp/wcs/stores/servlet/'
+const loginUrl = baseUrl + 'BLAuthentication'
+const billsUrl =
+  baseUrl + 'BLAccountOrdersHistoryCmd?purchase=allYear&store=store-site'
 
 module.exports = new BaseKonnector(start)
 
 function start(fields) {
-  // The BaseKonnector instance expects a Promise as return of the function
-  return request(`${baseUrl}/index.html`).then($ => {
-    // cheerio (https://cheerio.js.org/) uses the same api as jQuery (http://jquery.com/)
-    // here I do an Array.from to convert the cheerio fake array to a real js array.
-    const entries = Array.from($('article')).map(article =>
-      parseArticle($, article)
-    )
-    return addData(entries, 'com.toscrape.books').then(() =>
-      saveFiles(entries, fields)
-    )
-  })
+  log('info', 'Authenticating ...')
+  return authenticate(fields.email, fields.password)
+    .then(getList)
+    .then(entries => {
+      return saveBills(entries, fields, {
+        timeout: Date.now() + 60 * 1000,
+        identifiers: ['boulanger']
+      })
+    })
 }
 
-function authenticate(login, password) {
+function getList() {
+  return request({
+    method: 'GET',
+    uri: billsUrl
+  }).then(parseList)
```

Comment:
I believe you could simply write `request(billsUrl)`, GET is the default and 1st arg can be either the URL or an options object (idem, no need to fix it right now).